cypress/integration/play_word_and_card_spec.js

```diff
@@ -1,25 +1,47 @@
-describe('Play word and card', () => {
-    beforeEach(() => {
-        cy.login('unicorn')
-        cy.createRoom();
-        cy.startGame();
-    });
+const url = Cypress.config().baseUrl;
 
+describe('Play word and card', () => {
     describe('on start game', () => {
+        beforeEach(() => {
+            cy.login('unicorn');
+            cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(url)}`).then(() => {
+                cy.request(`http://localhost:12346/createRoom?url=${encodeURIComponent(Cypress.config().baseUrl)}`).then(() => {
```

Comment:
Could chain here as well. It's really a stylistic decision, whatever you think is more readable.